scripts/integration_test.sh

```diff
@@ -18,35 +18,36 @@ set -e
 
 # Runs integration tests on a given runtime image
 
-dir=`dirname $0`
-projectRoot=$dir/..
-testAppDir=$projectRoot/tests/test-war-integration
-deployDir=$testAppDir/target/docker-src
+readonly dir=$(dirname $0)
+readonly projectRoot=${dir}/..
+readonly testAppDir=${projectRoot}/tests/test-war-integration
+readonly deployDir=${testAppDir}/target/docker-src
+readonly service="tomcat-runtime-integration"
```

Comment:
I think this is OK to be hard-coded like this. IMO, integration tests should rarely be initiated manually, so as long as it uses a unique service, I think it's OK.